### DISCUSSION TITLE
MuJoCo Event at CoRL 2023

### DISCUSSION THREAD

**Discussion by nimrod-gileadi** [google-deepmind/mujoco#1140] [Announcements]
Hello, friends!

Members of the MuJoCo development team will be at CoRL 2023, and we want to invite you all to a meet-up on Monday, November 6th from 6:30-8:30pm ET at the Google Atlanta office. 

Please RSVP using [this form](https://rsvp.withgoogle.com/events/mujoco-corl-2023). We have capacity to host around 100 guests and cannot guarantee that everyone who registers will be able to attend. An invitation email will be sent to those who we are able to accommodate a few days before the event. Light refreshments will be provided!

Looking forward to seeing you there!

**Comment by goncamateus** [comment]
Hi there! Any chance to stream the meet-up?
